tests/script-based-pre/playback_expected/playback.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# This will run Kani verification in every `src/*.rs` file followed by playback command.
+# Expected output is generated from individual expected files.
+set -o nounset
```

Comment:
`-u` and `nounset` is the same thing. Since this script is running multiple tests and it's output is compared against the expected file, I prefer not failing if any command fails so it always runs all tests

Reply:
Note that Kani will return non-zero if the verification fails.